### PR TITLE
Propose a Basic Solution to Adjacency List Problem

### PR DESCRIPTION
**Pull request by StanimalTheMan**
- [ x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x ] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x ] My pull request targets the `master` branch of freeCodeCamp.
- [x ] None of my changes are plagiarized from another source without proper attribution.
- [x ] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x ] My changes do not use shortened URLs or affiliate links.

Closes #XXXXX

I am not sure my environment is 100% stable and when I tried to run it on local environment, it said to create  a React component to get the page to show up?  Nevertheless, I did the challenge on my own fcc account and it passed?  Sorry.

### CODE REVIEW

File: guide/english/certifications/coding-interview-prep/data-structures/adjacency-list/index.md [hunk 1]
```diff
@@ -2,9 +2,52 @@
...
+## ![:beginner:](https://forum.freecodecamp.com/images/emoji/emoji_one/beginner.png?v=3 ":beginner:") Basic Code Solution:
+
+    var undirectedAdjList = {
+        James: ["Jeff"],
+        Jill: ["Jenny"],
+        Jenny: ["Jill", "Jeff"],
+        Jeff: ["Jenny", "James"]
+    };
+
+![:rocket:](https://forum.freecodecamp.com/images/emoji/emoji_one/rocket.png?v=3 ":rocket:") <a href='https://repl.it/repls/SereneScholarlyAnalyst' target='_blank' rel='nofollow'>Run Code</a>
```
Comment (RandellDawson) [inline guide/english/certifications/coding-interview-prep/data-structures/adjacency-list/index.md]: Remove this line.

**Comment by RandellDawson** [comment]
When running locally, only the mock-guide is used, so to test a specific guide article located in the certifications folder, you would need to temporarily copy the the file you modified from `guide/english/certifications/coding-interview-prep/data-structures/adjacency-list/index.md` and put it in `mock-guide/english/certifications/coding-interview-prep/data-structures/adjacency-list/index.md` and then run `npm run develop` to see what it will actually look like.  

Just make sure you do not add the mock-guide file to your changes and push accidentally.

**Comment by StanimalTheMan** [comment]
Alright.  Thank you.

**Comment by thecodingaviator** [comment]
@StanimalTheMan If I understand correctly, you're asking if we want the stubs to be updated and have content and solutions. If that's the question, yes we do.

Please also have a look at the [Contributing docs](https://github.com/freeCodeCamp/freeCodeCamp/tree/master/docs) for the repo if you haven't already

**Review by thecodingaviator — APPROVED** [review]
LGTM! 👍

**Comment by thecodingaviator** [comment]
Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 📝

**Comment by StanimalTheMan** [comment]
Alright.  Thank you.
@thecodingaviator I went back as an user and pressed 'Get Hint', and there is still a stub. What exactly does this mean?

**Comment by RandellDawson** [comment]
@StanimalTheMan The change has only been made to the `master` branch.  It will be a little while before `master` is deployed to production.

**Pull request merged**